packages/now-build-utils/src/lambda.ts

```diff
@@ -105,3 +133,21 @@ export async function createZip(files: Files): Promise<Buffer> {
 
   return zipBuffer;
 }
+
+export async function getLambdaOptionsFromFunction(
+  sourceFile: string,
+  config?: Config
```

Comment:
Let's change the input parameters to an object like `{sourceFile: string, config?: Config}`.

I found out the hard way that you can never change `@now/build-utils` public APIs so this will make it easier in the future if we need to add new properties.

Reply:
Good point 👍